### PR TITLE
Add support for IS NOT NULL

### PR DESCRIPTION
**Pull request by yesdevnull**
Allows for correct `'value' != null` comparisons in SQL.

I've tested this with MSSQL but the syntax for MySQL looks to be the same.

### CODE REVIEW

**Comment by Naktibalda** [comment]
Please add tests for IS NULL and IS NOT NULL to https://github.com/Codeception/module-db/blob/master/tests/unit/Codeception/Lib/Driver/DbTest.php

File: src/Codeception/Lib/Driver/Db.php [hunk 1]
```diff
@@ -215,7 +215,12 @@ protected function generateWhereClause(array &$criteria)
         $params = [];
         foreach ($criteria as $k => $v) {
             if ($v === null) {
-                $params[] = $this->getQuotedName($k) . " IS NULL ";
+                if (stripos($k, ' !=') > 0) {
+                    $params[] = $this->getQuotedName(str_ireplace(" !=", '', $k)) . " IS NOT NULL ";
```
Comment (Naktibalda) [inline src/Codeception/Lib/Driver/Db.php:219]: `strpos` and `str_replace` works well enough for `!=` because there is only one way to write it.

**Review by Naktibalda — CHANGES_REQUESTED** [review]

File: src/Codeception/Lib/Driver/Db.php (hunk 1 above)
Comment (yesdevnull) [reply src/Codeception/Lib/Driver/Db.php:219]: Whoops, force of habit. I'll fix it up.

**Comment by yesdevnull** [comment]
Requested changes and tests done, back to you @Naktibalda.

**Comment by Naktibalda** [comment]
Thanks.

Would it be too much to ask you to document operators in Query Generation section?
https://github.com/Codeception/module-db/blob/master/src/Codeception/Module/Db.php#L198-L231

**Comment by yesdevnull** [comment]
@Naktibalda documentation added. Let me know if you want it fleshed out further.

**Comment by Naktibalda** [comment]
Thanks.

I looked at documentation page again and found that Supported operators are documented under [seeInDatabase](https://codeception.com/docs/modules/Db#seeInDatabase), [dontSeeInDatabase](https://codeception.com/docs/modules/Db#dontSeeInDatabase) and [grabFromDatabase](https://codeception.com/docs/modules/Db#grabFromDatabase), but not under [seeNumRecords](https://codeception.com/docs/modules/Db#seeNumRecords) and [grabNumRecords](https://codeception.com/docs/modules/Db#grabNumRecords).

> Comparison expressions can be used as well:
>
> ```php
> <?php
> $post = $I->grabFromDatabase('posts', ['num_comments >=' => 100]);
> $user = $I->grabFromDatabase('users', ['email like' => 'miles%']);
> ```
>
> Supported operators: `<`, `>`, `>=`, `<=`, `!=`, `like`.

I think that this section could be moved to Query Generation and a link to Query Generation should be added to docblock of each relevant method - [Comparison expressions can be used as well](#Query-generation).

What do you think?

Also `Since version 2.1.9` is no longer relevant.
